### DISCUSSION TITLE
Missing project icon in Vercel dashboard

### DISCUSSION THREAD

**Discussion by jessedelira** [vercel/community#7110] [Help]
### Summary

I set the image of a project in my vercel dashboard and now it has been showing as a placeholder for the past month. I'm not sure how to fix it and can't seem to find the place where I set the icon for the project in the settings anymore, seems to have been removed.

I deleted my cache, didn't help.

<img width="1193" alt="image" src="https://github.com/vercel/community/assets/38845310/0a228ab0-17d4-4d07-ba26-814a044fb234">

### Steps to Reproduce

I log into Vercel and click on the project, when I hit the dashboard I notice in the top left next to the project name there is no project image anymore.

**Comment by amyegan** [comment]
Hey @jessedelira. That isn't something you can set from the dashboard. The avatar should be the favicon from the project.

Is the favicon working correctly on your deployed site?

**Comment by jessedelira** [comment]
Hi @amyegan! Yes, my favicon is working as expected on my deployed [site](https://gym-tracker.app). I thought that was a icon that was set, thank you for letting me know that it is the favicon. I doubled checked to make sure that the favicon works and it does.
